### ISSUE TITLE
Ability to run individual test where test is part of a wider test suite

### ISSUE THREAD

**Issue by haroldb1** [zio/zio-intellij#439]
We would like to use the ability to play individual tests where they are part of a test suite that has been made accross separate files.
e.g. we have a test suite:

```
object TestSuite1 {
  val spec = suite("Test1")(
    test("test something) {
        ...
    }
  )
}
```

then in another file:
```
object WorkerHandlerSpec extends ZIOSpecDefault {

  val spec = suite("SPecificSpecs")(
    TestSuite1.spec,
    ...
  ) @@ TestAspect.silentLogging
}
```

**Comment by myazinn** [comment]
Yeah, that's somethings that has bothering us for a while now (e.g. https://github.com/zio/zio-intellij/issues/293)
The problem is that it actually requires support from both library and IDE plugin. So to support your use case, we need zio-test to be able to run arbitrary test cases (by name / regexp / whatever), and then we'll be able to wire it on our side.
I hope eventually we'll do it, but it'll definitely take some time

**Comment by haja** [comment]
I have a similar test setup, support for this would be awesome!
